main.js

```diff
@@ -56,7 +179,76 @@ function createEachProduct(products) {
 
 function renderPhotos(products) {
   for (var i = 0; i < products.length; i++) {
-    var $product = createEachProduct(products[i])
+    var $product = createEachProduct(products[i], i)
```

Comment:
Why pass `i` in as productId, `product[i]` has an `id` property which you should use as the product id.